.ado/pipelines/config/configuration.yaml

```diff
@@ -17,7 +17,7 @@ variables:
 - name: 'dotnetSdkVersion'    # dotnet sdk version
   value: '6.0.400'
 - name: 'uiAppNodeVersion'    # nodejs version for the UI app
-  value: '16.x'
+  value: '18.11'
```

Comment:
```suggestion
  value: '18.x'
```

Do we want to switch back to `18.x` here now that we know that it works?

Reply:
I'm fine either way. Let's just have it consistent with Online. Usually pinning a version is probably better - but then we need to take care of updates...

Reply:
I agree. Have no preference besides consistency. ;) @msimecek thoughts?

Reply:
Pin it!

Reply:
ok, then we leave it as is